deeppavlov/configs/ranking/ranking_insurance_interact.json

```diff
@@ -0,0 +1,127 @@
+        "out": ["x_proc"],
+        "sent_vocab":
+        {
+          "id": "siam_sent_vocab",
+          "name":"simple_vocab",
+          "save_path": "insurance_vocabs/sent.dict",
+          "load_path": "insurance_vocabs/sent.dict"
+        },
+        "tokenizer":
+        {
+          "name": "split_tokenizer"
+        },
+        "vocab":
+        {
+          "id": "siam_vocab",
+          "name":"simple_vocab",
+          "save_path": "insurance_vocabs/tok.dict",
+          "load_path": "insurance_vocabs/ctok.dict"
+        },
+        "embedder":
+        {
+          "id": "siam_embedder",
+          "name": "fasttext",
+          "load_path": "insurance_embeddings/wiki.en.bin",
+          "save_path": "insurance_embeddings/wiki.en.bin",
+          "dim": 300
+        }
+      },
+      {
+        "id": "embeddings",
+        "name": "emb_mat_assembler",
+        "embedder": "#siam_embedder",
+        "vocab": "#siam_vocab"
+      },
+      {
+        "id": "model",
+        "name": "bilstm_nn",
+        "len_vocab": "#siam_vocab.len",
+        "use_matrix": "#preproc.use_matrix",
+        "max_sequence_length": "#preproc.max_sequence_length",
+        "emb_matrix": "#embeddings.emb_mat",
+        "embedding_dim": "#siam_embedder.dim",
+        "seed": 243,
+        "reccurent": "bilstm",
+        "max_pooling": true,
+        "shared_weights": true,
+        "hidden_dim": 300,
+        "learning_rate": 1e-3,
+        "triplet_loss": true,
+        "hard_triplets": false,
+        "margin": 0.1,
+        "batch_size": 256,
+        "save_path": "insurance_model/model_weights.h5",
+        "load_path": "insurance_model/model_weights.h5",
+        "preprocess": "#preproc.__call__",
+        "interact_pred_num": 3
+      },
+      {
+        "in": ["x_proc"],
+        "in_y": ["y"],
+        "out": ["y_predicted"],
+        "name": "siamese_predictor",
+        "model": "#model",
+        "batch_size": "#model.batch_size",
+        "interact_pred_num": 3,
+        "responses": "#siam_sent_vocab",
+        "preproc_func": "#preproc.__call__"
+      }
+    ],
+    "out": ["y_predicted"]
+  },
+  "train": {
+    "epochs": 200,
+    "batch_size": 256,
+    "pytest_max_batches": 2,
+    "train_metrics": [],
+    "metrics": ["r@1_insQA", "rank_response"],
+    "validation_patience": 5,
+    "val_every_n_epochs": 5,
+    "log_every_n_batches": 24
+  },
+  "metadata": {
+    "requirements": [
+      "../dp_requirements/tf.txt",
+      "../dp_requirements/gensim.txt"
+    ],
+    "labels": {
+      "telegram_utils": "SiameseModel",
+      "server_utils": "Ranker"
+    },
+    "download": [
+      "http://files.deeppavlov.ai/deeppavlov_data/insurance_ranking.tar.gz",
+      {
+        "url": "http://files.deeppavlov.ai/datasets/insuranceQA-master.zip",
+        "subdir": "insurance_data"
+      },
+      {
+        "url": "http://files.deeppavlov.ai/deeppavlov_data/embeddings/wiki.en.bin",
+        "subdir": "insurance_embeddings"
```

Comment:
Would it be ok to use `embeddings` subdirectory for `wiki.en.bin`? This file is used for other models and it's wasteful to have its copies in `embeddings`, `insurance_embeddings` and `ubuntu_v2_embeddings`

Reply:
fixed

Reply:
the space on disk is not wasted but embeddings are still downloaded multiple times